lib/fine-types/test/data/Cardano/Ledger/Alonzo/Tx.fine

```diff
@@ -0,0 +1,92 @@
+module Tx where
+
+import Address(Addr, Addr_rwd, Ix, Slot, Network);
+import Crypto(VKey, Sig, ScriptHash, Script, KeyHash_G, DataHash, KeyHash);
+import PParams(Coin, Epoch, PParamsUpdate);
+import Delegation(DCert);
+import Script(Script^ph1);
+import Value(Value);
+import CostModel(ExUnits);
+
+{-----------------------------------------------------------------------------
+    Transactions
+    Shelley spec, Figure 10
+    Mary spec, Figure 5
+    Alonzo spec, Figure 2
+------------------------------------------------------------------------------}
+-- Abstract types
+
+ScriptIntegrityHash         = _; --^ Script data hash
+Script_plc                  = _; --^ Plutus core scripts
+Data                        = _; --^ Generic script data
+TxId                        = _ ; --^ transaction identifier
+Metadatum                   = _ ; --^ metadatum
+AuxiliaryDataHash           = _ ; --^ auxiliary data hash
+
+-- Script types
+
+Script^ph2                  = Script_plc;
+Script                      = Script^ph1 + Script^ph2;
+IsValid                     = Bool;
+Datum                       = Data;
+Redeemer                    = Data;
+
+-- Derived types
+
+ValidityInterval            = Slot? × Slot?;
+Tag                         =
+    Σ{Spend                 : Unit
+    , Mint                  : Unit
+    , Cert                  : Unit
+    , Rewrd                 : Unit
+    };
+RdmrPtr                     = Tag × Ix;
+TxIn                        = TxId × Ix; --^ transaction input
+TxOut                       = Addr × Value × DataHash?; --^ transaction output
+UTxO                        = TxIn ↦ TxOut; --^ unspent tx outputs
+Wdrl                        = Addr_rwd ↦ Coin; --^ reward withdrawal
+Metadata                    = ℕ ↦ Metadatum; --^ metadata
+
+-- Derived types (update system)
+
+ProposedPPUpdates           = KeyHash_G ↦ PParamsUpdate; --^ proposed updates
+Update                      = ProposedPPUpdates × Epoch; --^ update proposal
+
+-- Transaction Types
+
+TxBody                      =
+    { txins                 : ℙ TxIn  --^ transaction inputs
+    , collateral            : ℙ TxIn  --^ collateral inputs
+    , txouts                : Ix ↦ TxOut  --^  transaction outputs
+    , txcerts               : DCert*  --^ delegation certificates
+    , mint                  : Value  --^ value minted in this transaction
+    , txfee                 : Coin  --^ transaction fee
+    , txvldt                : Slot? × Slot?  --^ validity interval
```

Comment:
```suggestion
    , txvldt                : ValidityInterval  --^ validity interval
```

I just noticed this — they are using the named type since Alonzo. 🤷